google/cloud/storage/doc/storage-main.dox

```diff
@@ -148,6 +148,27 @@ using gcs = ::google::cloud::storage;
 }
 @endcode
 
+## Overriding the default endpoint
+
+In some cases, you may need to override the default endpoint used by the client
+library. The `google::cloud::storage::RestEndpointOption` can be used in this
+case:
+
+@snippet storage_client_initialization_samples.cc set-client-endpoint
+
+## Override the authentication configuration
```

Comment:
Perhaps `s/Override/Overriding/` to match above, or vice versa?

Reply:
Fixed.